### PR TITLE
Topic invite modal

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge?

File: api/models/CommunityTag.js [hunk 1]
```diff
@@ -20,5 +20,9 @@ module.exports = bookshelf.Model.extend({
     .where({community_id: communityId, tag_id: tagId})
     .count()
     .then(rows => Number(rows[0].count))
+  },
+
+  findParticipants (communityId, tagId, limit) {
```
Comment (levity) [inline api/models/CommunityTag.js]: ?

File: test/unit/controllers/InvitationController.test.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var factories = require(root('test/setup/factories'))
 var InvitationController = require(root('api/controllers/InvitationController'))
 import { mockify, unspyify } from '../../setup/helpers'
 
-describe('InvitationController', () => {
+describe.only('InvitationController', () => {
```
Comment (levity) [inline test/unit/controllers/InvitationController.test.js]: here's a stray `only`

File: api/models/Invitation.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ module.exports = bookshelf.Model.extend({
       let creator = invitation.relations.creator
       let community = invitation.relations.community
 
-      let data = _.extend(_.pick(opts, 'message', 'subject'), {
+      let data = _.extend(_.pick(opts, 'message', 'subject', 'participants'), {
```
Comment (levity) [inline api/models/Invitation.js]: want to forward me an example of an email with participants?

**Comment by levity** [comment]
looks good otherwise

**Pull request merged**